tests/phpunit/Compat/WC_Stripe_Subscription_Initial_Test.php

```
@@ -0,0 +1,141 @@
+								->disableOriginalConstructor()
+								->onlyMethods( [ 'get_upe_enabled_at_checkout_payment_method_ids' ] )
+								->getMock();
+
+		// Mock the UPE method to return card payment method
+		$wc_gateway_stripe
+			->expects( $this->any() )
+			->method( 'get_upe_enabled_at_checkout_payment_method_ids' )
+			->will( $this->returnValue( [ WC_Stripe_Payment_Methods::CARD ] ) );
+
+		// Mock the payment methods to include the card method, which is required for the test to run.
+		$wc_gateway_stripe->payment_methods = [ WC_Stripe_Payment_Methods::CARD => new \WC_Stripe_UPE_Payment_Method_CC() ];
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '47', column '82'). (undefined)

(MissingImport)